src/logic/safe/transactions/awaitingTransactions.js

```diff
@@ -4,6 +4,7 @@ import type { Transaction } from '~/routes/safe/store/models/transaction'
 
 export const getAwaitingTransactions = (
   allTransactions: Map<string, List<Transaction>>,
+  cancelTransactionsByNonce: Map<string | number, List<Transaction>>,
```

Comment:
can this be renamed to `cancellationTxsByNonce` maybe? At first I though this was a list of transactions to cancel by nonce